lib/python/picongpu/plugins/data/transitionradiation.py

```diff
@@ -0,0 +1,274 @@
+                raise ValueError("Invalid index for Phi!")
+        if omega is not None:
+            omega = int(omega)
+            if omega < 0 or omega >= len(self.omegas):
+                raise ValueError("Invalid index for Omega!")
+
+        if type == "spectrum":
+            # find phi and theta with maximum intensity if they are not given as parameters
+            if theta is None and phi is None:
+                maxIndex = np.argmax(self.data[:, :], 0)
+                theta = int(np.floor(maxIndex[0] / len(self.phis)))
+                phi = maxIndex[0] % len(self.phis)
+            elif theta is None and phi is not None:
+                theta = np.argmax(self.data[phi::len(self.phis), :], 0)[0]
+            elif theta is not None and phi is None:
+                phi = np.argmax(self.data[theta * len(self.phis):(theta + 1) * len(self.phis):, :], 0)[0]
+
+            print("Spectrum is plotted at phi={:.2e} and theta={:.2e}".format(self.phis[phi], self.thetas[theta]))
+            return self.omegas, self.data[theta * len(self.phis) + phi, :]
+        elif type == "sliceovertheta":
+            # find phi and omega with maximum intensity if they are not given as parameters
+            if omega is None and phi is None:
+                maxIndex = np.argmax(self.data[:, :], 0)
+                phi = maxIndex[0] % len(self.phis)
+                omega = 0
+            if omega is not None and phi is None:
+                maxIndex = np.argmax(self.data[:, omega])
+                phi = maxIndex % len(self.phis)
+            if omega is None and phi is not None:
+                omega = 0
+
+            print("Angular intensity distribution is sliced at phi={:.2e} "
+                  "with omega={:.2e}.".format(self.phis[phi], self.omegas[omega]))
+            return self.thetas, self.data[phi::len(self.phis), omega]
+        elif type == "sliceoverphi":
+            # find theta and omega with maximum intensity if they are not given as parameters
+            if theta is None and omega is None:
+                maxIndex = np.argmax(self.data[:, :], 0)
+                theta = int(np.floor(maxIndex[0] / len(self.phis)))
+                omega = 0
+            if theta is not None and omega is None:
+                omega = 0
+            if theta is None and omega is not None:
+                maxIndex = np.argmax(self.data[:, omega])
+                theta = int(np.floor(maxIndex / len(self.phis)))
+
+            print("Angular intensity distribution is sliced at theta={:.2e} "
+                  "with omega={:.2e}.".format(self.thetas[theta], self.omegas[omega]))
+            return self.phis, self.data[theta * len(self.phis):(theta + 1) * len(self.phis), omega]
+        elif type == "heatmap":
+            # find omega with maximum intensity if it is not given as parameter
+            if omega is None:
+                omega = 0
+                print("theta: ", theta, "phi: ", phi, "omega: ", omega)
```

Comment:
I think here you do not need spaces after the colon - when separated by a comma, print adds a space.

Reply:
I'm pretty sure, this was just a debug message. I'll remove it